src/GitHub/Endpoints/Repos.hs

```diff
@@ -172,6 +173,12 @@ createRepoR :: NewRepo -> Request 'RW Repo
 createRepoR nrepo =
     command Post ["user", "repos"] (encode nrepo)
 
+-- | Fork an existing repository.
+-- See <https://developer.github.com/v3/repos/forks/#create-a-fork>
+forkExistingRepoR :: Name Owner -> Name Repo -> Maybe (Name Owner) -> Request 'RW Repo
+forkExistingRepoR owner repo morg =
+    command Post ["repos", toPathPart owner, toPathPart repo, "forks" ] (maybe mempty encode morg)
```

Comment:
Went the "TODO" route instead. The other option seems like a bit of a rabbit hole.